.github/workflows/lighthouse.yml

```diff
@@ -2,64 +2,25 @@ name: Lighthouse PWA Test
 
 on:
   pull_request:
-  #  branches:
-  #    - master
   push:
     branches: [ master, develop ]
 
 jobs:
-  lighthouse-local:
+  lighthouse:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - name: Build PWA distribution for production
```

Comment:
@midnight-madman should we leave in the lighthouse local static check as a quick validation step before testing against preview deployment?